docs/content/en/docs/references/feedback/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Skaffold Feedback"
+linkTitle: "Skaffold Feedback"
+weight: 300
+---
+
+Your feedback is very important to us to make Skaffold better!
+Sending us feedback is easy.
+
+## Quick HaTS Survey
+Take our [quick 5-question survey](https://forms.gle/BMTbGQXLWSdn7vEs6) to tell us how satisfied you are with
+Skaffold and and what improvements we should make.
+
+## Skaffold Feedback Sessions
+The Skaffold team actively incorporates user feedback as the product evolves.
+We are looking to chat with developers like you to learn about how you use Skaffold and
```

Comment:
maybe rephrase to something like:

We are looking to chat with developers like you to learn how you use Skaffold and how we can improve it to better suit your needs.